src/lib/DatasetTags.svelte

```diff
@@ -3,13 +3,14 @@
 	export let path = '/dataset';
 </script>
 
-<h3>Tags</h3>
-<div class="tags">
-	{#each dataset.tags as tag}
-		<a class="tag" href="{path}?tags={tag.name}">{tag.display_name}</a>&nbsp;
-	{/each}
-</div>
-
+{#if dataset.tags}
```

Comment:
I would move the Tags-heading out of the if-clause, so that it does show that there are generally tags, but this dataset just does not have one. I think that would make it more consistent. WDYT?